Source/Core/DolphinWX/InputConfigDiag.cpp

```diff
@@ -946,7 +946,14 @@ ControlGroupBox::ControlGroupBox(ControllerEmu::ControlGroup* const group, wxWin
 					setting->wxcontrol->Bind(wxEVT_SPINCTRL, &GamepadPage::AdjustSetting, eventsink);
 					options.push_back(setting);
 					wxBoxSizer* const szr = new wxBoxSizer(wxHORIZONTAL);
-					szr->Add(new wxStaticText(parent, wxID_ANY, wxGetTranslation(StrToWxStr(groupSetting->name))), 0, wxCENTER | wxRIGHT, 3);
+					wxStaticText* nameBox = new wxStaticText(parent, wxID_ANY, wxGetTranslation(StrToWxStr(groupSetting->name)));
+					if (groupSetting->name == "Battery" && !SConfig::GetInstance().m_DolphinPro)
+					{
+						setting->wxcontrol->Disable();
+						nameBox->SetToolTip(wxGetTranslation("Upgrade to Dolphin Pro� to recharge batteries."));
```

Comment:
Also `wxGetTranslation("Upgrade to Dolphin Pro to recharge batteries.")` should be `_("Upgrade to Dolphin Pro to recharge batteries.")`